app/shared/types/settingsSchema.ts

```diff
@@ -73,6 +73,8 @@ export const settingsLinkSchema = {
     _id: objectIdSchema,
     title: { type: 'string' },
     url: { type: 'string' },
+    sublinks: { type: 'array' },
```

Comment:
I thin this should be `sublinks: { type: 'array', items: { type: 'string' } }`
to avoid emitting as unknown

Reply:
Added